index.js

```diff
@@ -87,7 +95,11 @@ var gs = {
     if (positives.length === 0) throw new Error('Missing positive glob');
 
     // only one positive glob no need to aggregate
-    if (positives.length === 1) return streamFromPositive(positives[0]);
+    if (positives.length === 1) {
+      return streamFromPositive(positives[0]);
+    } else {
+      opt.allowEmpty = true;
```

Comment:
Uh, ignore what I just said. You said in the gulp issue that it should be _false_ by default (so, it throws an error if an option isn't specified). Have you changed your mind?